tests/subsys/pm/power_mgmt_multicore/src/main.c

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2021 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <kernel.h>
+#include <ztest.h>
+#include <ksched.h>
+#include <kernel.h>
+#include <pm/pm.h>
+
+
+#if CONFIG_MP_NUM_CPUS != 2
+#error "Invalid number of cpus"
+#endif
+
+#define NUM_OF_ITERATIONS 5
+
+#define THRESHOLD 10
+
+#define ACTIVE_MSEC 40
+#define ACTIVE_TIMEOUT K_MSEC(40)
+#define IDLE_MSEC (60 + THRESHOLD)
+#define IDLE_TIMEOUT K_MSEC(60)
+#define SUSPEND_TO_IDLE_MSEC (100 + THRESHOLD)
+#define SUSPEND_TO_IDLE_TIMEOUT K_MSEC(100)
+#define STANDBY_TIMEOUT K_MSEC(200)
```

Comment:
Adding docstrings to these contants would help to understand what test does/why these values

Reply:
They are totally arbitrary, since it is consumed entirely in here. Will add a comment about it though.